### ISSUE TITLE
Add an Extend time button/function OR an alert

### ISSUE THREAD

**Issue by severinmenard** [hotosm/osm-tasking-manager2#770]
When participating in a mapathon that lasts more than 2 hours, every contributor need to be really careful of the time running, in order to avoid someone else taking their task once the 2 hours pattern has been spent. 
Could be good to have:
- either a link proposing to someone who took a task to add some extra time. Would not hurt to propose a dropdown list allowing to get the full extent of the planned mapathon. Eg +3 hours for a mapathon planned between 10AM and 15PM, so that there is no risk of duplications. 
- or an alert box in the editor

**Comment by vossad01** [comment]
I think the two proposals listed complementary. Since the editor is separate from HOT the second option of an alert box in the editor would probably not be consistently be achievable.

However, the [`Web Notifications`](http://caniuse.com/#feat=notifications) may have just enough implementation/adoption to be considered to provide alert boxes.

**Comment by bgirardot** [comment]
How often do people work on a task more than two hours? It is rare circumstances when a single task square should take more than 2 hours to map. They should be consistently completable in 1 hour max, 15 minutes better.

**Comment by RAytoun** [comment]
I agree that two hours is sufficient. Tiles at Mapathons should be split to allow beginners to be able to complete them within 30 minutes so they do not get bored. More experienced mappers know to exit the tile and reopen it if they need to spend a lot of time on it.
